src/plugins/intel_cpu/tests/functional/custom/subgraph_tests/src/common/concat_multiple_query_sdp.cpp

```diff
@@ -290,17 +304,24 @@ class ConcatMultiQuerySDPTest : public testing::WithParamInterface<ConcatMultiQu
 TEST_P(ConcatMultiQuerySDPTest, CompareWithRefs) {
     SKIP_IF_CURRENT_TEST_IS_DISABLED();
     InputShapeAndTransposeOrder inputShapeAndOrders;
+    bool forceKVU8;
     bool hasShapeOf;
     ElementType qkvType;
-    std::tie(qkvType, inputShapeAndOrders, hasShapeOf) = this->GetParam();
-    if (qkvType == ElementType::bf16 && !ov::with_cpu_x86_bfloat16())
+    std::tie(qkvType, inputShapeAndOrders, forceKVU8, hasShapeOf) = this->GetParam();
+    if ((qkvType == ElementType::bf16 && !ov::with_cpu_x86_bfloat16()) ||
```

Comment:
Can we instans only valid test params combinations instead of implicit skips inside the test body?

Reply:
> Can we instans only valid test params combinations instead of implicit skips inside the test body?

It is a runtime aware testcase, especially for different ISA.
This test body will not run on unsupported platform. I think it it relates to ISA instead of params.

Reply:
We already have generic mechanism that skips all the bf16 tests on non avx512 platforms. https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp#L518-L523
So I don't understand why this test has to be implemented in different way

Reply:
> We already have generic mechanism that skips all the bf16 tests on non avx512 platforms. https://github.com/openvinotoolkit/openvino/blob/master/src/plugins/intel_cpu/tests/functional/shared_tests_instances/skip_tests_config.cpp#L518-L523 So I don't understand why this test has to be implemented in different way

I see. remove the skip codes.